bubblegum/program/src/utils.rs

```diff
@@ -91,14 +92,14 @@ pub fn append_leaf<'info>(
     let authority_pda_signer = &[&seeds[..]];
     let cpi_ctx = CpiContext::new_with_signer(
         gummyroll_program.clone(),
-        gummyroll::cpi::accounts::Modify {
+        spl_compression::cpi::accounts::Modify {
             authority: authority.clone(),
-            merkle_roll: merkle_roll.clone(),
-            candy_wrapper: candy_wrapper.clone(),
+            merkle_tree: merkle_roll.clone(),
+            log_wrapper: candy_wrapper.clone(),
```

Comment:
Nit the param is still called `candy_wrapper`, probably should be changed to `log_wrapper`?